src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$main-text-color: #fff;
+$main-bg-color: #1d1d1d;
+$button-text-color: #1d1d1d;
+$menu-text-color: #1d1d1d;
+$menu-bg-color: #c2c2c2;
+$defaut-form-text-color: #7e7e83;
+$user-form-text-color: #292929;
+$border-form-hover-color: #d5d7de;
+$effectDuration: 0.3s;
```

Comment:
It is better to name the variables in the same style.
```
$duration-effect: 0.3s;
```

Reply:
done